databases/week3/Meal sharing.sql

```diff
@@ -0,0 +1,211 @@
+-- Create all the sql for creating the data model
+
+set names utf8mb4;
+CREATE DATABASE MealSharing;
+
+CREATE TABLE meal (
+id int unsigned NOT NULL AUTO_INCREMENT,
+title varchar(255) NOT NULL,
+description TEXT,
+location varchar(255) NOT NULL,
+`when` DATETIME NOT NULL,
+max_reservations INT,
```

Comment:
Should max reservations be optional, ie. NULLable? Also consider making it unsigned since you don't need to store negative integers

Reply:
Thanks